### DISCUSSION TITLE
How can I achieve multi-line truncate

### DISCUSSION THREAD

**Discussion by thesobercoder** [tailwindlabs/tailwindcss#3179] [Help] [answered]
I have a specific need to truncate after 2 lines, I don't seem to find a way to do it Tailwind. Please help.

**Comment by adamwathan** [comment, marked as answer]
You'll probably need to write custom CSS or possibly even JS for this, here's a good article:

https://css-tricks.com/line-clampin/

**Comment by thesobercoder** [comment]
Thanks Adam. I found this plugin for now `tailwindcss-truncate-multiline`. Hoping that this will get baked into Tailwind someday.
